test/libsolidity/SemanticTest.cpp

```diff
@@ -425,7 +425,7 @@ TestCase::TestResult SemanticTest::runTest(
 			{
 				soltestAssert(
 					m_allowNonExistingFunctions ||
-					m_compiler.methodIdentifiers(m_compiler.lastContractName(m_sources.mainSourceFile)).isMember(test.call().signature),
+					m_compiler.methodIdentifiers(m_compiler.lastContractName(m_sources.mainSourceFile)).count(test.call().signature),
```

Comment:
```suggestion
					m_compiler.methodIdentifiers(m_compiler.lastContractName(m_sources.mainSourceFile)).contains(test.call().signature),
```

Also in other files.

Reply:
Done.